_targets.R

```diff
@@ -145,13 +139,20 @@ list(
       filter(geo_code1 %in% z$DataZone) |>
       filter(geo_code2 %in% z$DataZone)
     set.seed(2023)
+    # Test if cargo is available to system:
+    source("R/is_bin_on_path.R")
+    if (is_bin_on_path("odjitter")) {
+      old_path = Sys.getenv("PATH")
+      Sys.setenv(PATH = paste(old_path, "/root/.cargo/bin", sep = ":"))
+    }
     odj = odjitter::jitter(
       od = od,
       zones = z,
       subpoints_origins = subpoints_origins,
       subpoints_destinations = subpoints_destinations,
       disaggregation_threshold = 30,
-      deduplicate_pairs = FALSE
+      deduplicate_pairs = FALSE,
+      odjitter_location = "/root/.cargo/bin/odjitter"
```

Comment:
To remove.

Reply:
quick one: These lines need to stay, removing it will cause the odjitter to miss again.